cdap-metadata-spi/src/main/java/io/cdap/cdap/spi/metadata/LineageInfo.java

```diff
@@ -199,16 +199,16 @@ public Builder addTargets(Set<Asset> targets) {
     /**
      * Set the start time of the program run.
      */
-    public Builder setStartTime(Long startTime) {
-      this.startTime = startTime;
+    public Builder setStartTimeMs(Long startTimeMs) {
```

Comment:
existing issue, but the argument should be annotated with `@Nullable` if it is allowed to be null. If it is not supposed to be nullable, then it should be a long instead of Long.